examples/node/lib/rest/api/v2010/account/call/feedbackCallSummary.ts

```diff
@@ -226,7 +226,7 @@ export class FeedbackCallSummaryInstance {
   sid?: string | null;
   testString?: string | null;
   testInteger?: number | null;
-  testObject?: object | null;
+  testObject?: PhoneNumberCapabilities | null;
```

Comment:
This needs to be imported. The config file says where it's located.

Reply:
I did this by adding it directly to api-single.mustache. This will add the import into some files that don't use it, but the linter will remove all unused imports. I stole this approach from Java. 

Is this acceptable? Or do we want some sort of logic to only add the imports to necessary files?

Reply:
I'd prefer to programmatically include the specific import needed (or all the imports listed in the config file) so that we we need not touch the template file when new types are later added.